Homework_2.playground/Contents.swift

```diff
@@ -68,18 +138,29 @@ class StackStatistics: Stack {
         stackOfMinElements.push(element: currentMinElement)
     }
     
-    override func pop() -> Int? {
+    override func pop() -> Element? {
         guard let element = super.pop() else { return nil }
         stackOfMinElements.pop()
         return element
     }
     
-    func minimumElement() -> Int? {
+    /// Return minimum element in the stack
```

Comment:
```suggestion
    /// Returns minimum element in the stack
```

You use present simple in the rest of doc comments.

Reply:
Done.